administration/src/cards/Card.ts

```diff
@@ -131,10 +131,28 @@ export const isExpirationDateValid = (card: Card, { nullable } = { nullable: fal
   )
 }
 
-export const isValid = (card: Card, { expirationDateNullable } = { expirationDateNullable: false }): boolean =>
-  isFullNameValid(card) &&
-  getExtensions(card).every(({ extension, state }) => extension.isValid(state)) &&
-  (isExpirationDateValid(card, { nullable: expirationDateNullable }) || hasInfiniteLifetime(card))
+export const cardHasAllMandatoryExtensions = (card: Card, cardConfig: CardConfig): boolean => {
+  const mandatoryExtensions = cardConfig.extensions.filter(extension => extension.isMandatory)
+  return mandatoryExtensions.every(extension => Object.keys(card.extensions).includes(extension.name))
```

Comment:
:thinking: I guess our current architecture prevents it, but in theory it would be possible that the value is still null or undefined as you are only checking the key, which would not really count as `has all mandatory extensions` imo. Works fine atm though.